tests/posix/signal/testcase.yaml

```
@@ -0,0 +1,31 @@
+common:
+  filter: not CONFIG_NATIVE_LIBC
+  tags:
+    - posix
+    - signal
+  # 1 tier0 platform per supported architecture
+  platform_key:
+    - arch
+    - simulation
+tests:
+  portability.posix.signal:
```

Test cases should reflect the option group name 
```suggestion
  portability.posix.signals:
```